internal/kv/rocksdb/rocksdb_kv.go

```diff
@@ -199,19 +199,13 @@ func (kv *RocksdbKV) LoadBytesWithPrefix(prefix string) ([]string, [][]byte, err
 
 // MultiLoad load a batch of values by keys
 func (kv *RocksdbKV) MultiLoad(keys []string) ([]string, error) {
-	if kv.DB == nil {
-		return nil, errors.New("rocksdb instance is nil when do MultiLoad")
+	valuesInBytes, err := kv.MultiLoadBytes(keys)
```

Comment:
prefer to implement this with `kv.DB.MultiGet()`, cause call `kv.MultiLoadBytes()` directly which will create much more temporary  bytes objects

Reply:
@weiliu1031 Yes, the code author should also have this concern. But the original implementation also converted the byte array to the string, so I think it's fine to remove the duplicate code.

Reply:
1. notice origin line-223 in `kv.MultiLoadBytes()`, it create a temporary bytes object as return value. 
![image](https://github.com/milvus-io/milvus/assets/108661493/9139f515-a7c7-497c-b124-c4e914fc5323)

2. in `kv.MultiLoad()`, we can call `kv.DB.MultiGet()` directly, and then convert `value.Data()` to string, which will avoid the non-necessary temporary bytes in `kv.MultiLoadBytes()`

Reply:
@weiliu1031 Thanks for your information, I got your point now. So do you think I should revert this change?

Reply:
> @weiliu1031 Thanks for your information, I got your point now. So do you think I should revert this change?

this pr is still in hold status, you can update your commit, and we will merge it, it a good job!

Reply:
done